applications/HexPlay.md

```diff
@@ -157,7 +157,7 @@ The primary back-end development will be implemented in rust programming languag
 | 6.     | Testing                | The code will have proper unit-test coverage to ensure functionality and robustness. |
 | 7.     | Docker                 | We will provide a docker file to demonstrate the functionality of our chain with the corresponding module. |
 
-### Mildestone 2: Staking-Mining module, Distribution Module, and their Front-End
+### Mildestone 2: Staking-Mining module, and their Front-End
 * **Estimated Duration:** 1 months
 * **FTE:**  4
 * **Costs:** 0.3 BTC
```

Comment:
That's quite expensive for one module that now forms this milestone. Can you at least halve it please?

Reply:
We are thinking about merger M1 and M2, and require 2 month of time, and a total of 0.5 BTC. will this look good? Do you have any idea on how to group the work together? Thanks a lot.

Reply:
Merging sounds reasonable to me.

> Do you have any idea on how to group the work together? 

What do you mean by this? You can edit the application file and re-arrange the deliverables from M2 to M1, thus effectively deleting M2.